pkg/storage/gcp/gcp.go

```diff
@@ -0,0 +1,68 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"cloud.google.com/go/datastore"
+	"cloud.google.com/go/storage"
+	"github.com/gomods/athens/pkg/config/env"
+	"google.golang.org/api/option"
+)
+
+// Storage implements the following interfaces:
+// (./pkg/storage).Saver
+// (./pkg/storage).Getter
+// (./pkg/storage).Lister
+// (./pkg/storage).Checker
+// (./pkg/storage).Deleter
+type Storage struct {
+	bucket       *storage.BucketHandle
+	dsClient     *datastore.Client
```

Comment:
Is it used somewhere?. I coudn't find anything.

Reply:
I can't count how many times I've removed this :smiley: It keeps coming in from master

Reply:
maybe that's a lie, not master. but I have been having some 'issues' merging master. I should probably just wait until I have actual free time to work on it instead of small bits.